server/modules/scheduling/index.js

```diff
@@ -159,25 +195,27 @@ function getScheduleForInstance(instance) {
 
 function parseEnvironmentSchedule(environmentSchedule) {
   if (environmentSchedule.ManualScheduleUp === false && environmentSchedule.ScheduleAutomatically === false) {
```

Comment:
When two things like this act as a single predicate, I really like a single method here to explain what it means when these two things are the case. What single check are we performing here? *minor thing*